requirements.txt

```diff
@@ -0,0 +1,17 @@
+python>=3.9
+
+geopandas>=0.11.1
```

Comment:
Pandas 1.0 [required dependency for GeoPandas](https://geopandas.org/en/stable/getting_started/install.html)? 
(I'm assuming installing geopandas>=0.11.1 will install all required dependencies { inc. pandas 1.0 })